packages/xstate-react/test/useActor.test.tsx

```diff
@@ -13,10 +13,12 @@ import {
 } from 'xstate';
 import { useMachine } from '../src';
 import { useActor } from '../src/useActor';
-import { describeEachReactMode } from './utils';
+import { describeEachReactMode, once } from './utils';
 
 describeEachReactMode('useActor (%s)', ({ render }) => {
   it('initial invoked actor should be immediately available', (done) => {
+    const onDone = once(done);
```

Comment:
This is kind of annoying to have to do, but `done` is being called twice because of `React.StrictMode` behavior. This was always the case, but calling `done` multiple times is a failure in newer versions of `jest`